src/dq_suite/common.py

```diff
@@ -172,9 +172,9 @@ def write_to_unity_catalog(
     full_table_name = get_full_table_name(
         catalog_name=catalog_name, table_name=table_name
     )
-    df.write.mode(mode).option("overwriteSchema", "true").saveAsTable(
-        full_table_name
-    )  # TODO: write as delta-table? .format("delta")
+    df.write.format("delta").mode(mode).option(
```

Comment:
Ik snap de behoefte, maar dit wordt een lastig puntje wanneer we de opslag van resultaten willen gaan centraliseren. Ben benieuwd naar jouw gedachten hierover. Keertje meeten?

Reply:
Ja laten we dit komende week even bespreken, ik ben wel benieuwd waar je moeilijkheden ziet.

Reply:
Ik ga kijken of we de functionaliteit van delta-spark kunnen vangen in de PySpark API. Dan zouden we zonder het stukje delta-spark code toekunnen, wat het geheel simpeler maakt.